ontology/yaml/resources/HVAC/entity_types/CMP.yaml

```diff
@@ -38,3 +38,13 @@ CMP_SS_WDPM:
   - run_mode
   - schedule_run_command
   - control_status
+
+CMP_SS_ZTO:
+  description: "Basic features of air compressor with zone temp override."
+  is_canonical: true
+  uses:
+  - zone_override_temperature_setpoint
```

Comment:
What does this override temperature setpoint mean, and where is the associated setpoint? I am unclear how a compressor does any sort of zone temperature control, which is what this field implies. Please clarify.

Reply:
Hi sir, We use this as ACU (CMP) front to several FCUs

![ACU_FCU](https://user-images.githubusercontent.com/21256796/165198368-7509bf79-a5eb-4e68-9fbf-a8efec2537dd.png)

I'm asking my engineer colleague if there is specific FCU that this ACU setpoint controls. We may not use it if it's unclear (moving it to opt_uses) but as per specs, this is what it has as its points list (SS, on/off status and this zone override temp setpoint).

Reply:
Hi again sir,

I was wrong, document we have as points list has typo: ACU has only SS (not including zone temp override setpoint). the zone temp override setpoint got a corrputed header it is part of the occupancy sensor (which has been removed now) and so the point list under occupancy sensor (in excel) got into the group above it when occupancy header got removed. I'll revert this ASAP.

Thanks for catching it sir, I am not really accustomed to hardwares like this, as I am a software developer by trade hehe.